include/nuSQuIDS/body.h

```diff
@@ -762,6 +766,123 @@ class EarthAtm: public Body{
     Track MakeTrackWithCosine(double cosphi);
 };
 
+
+
+/// \class EmittingEarthAtm
+/// \brief A model of the production of neutrino cosmic ray flux production within the atmosphere.
+class EmittingEarthAtm: public EarthAtm{
+  protected:
+    /// \brief Atm differential nu electron flux array
+    std::vector<double> diff_enu_prod;
+    marray<double,1> enu_flux;
+    /// \brief Atm differential nu muon  flux array
+    std::vector<double> diff_munu_prod;
+    marray<double,1> munu_flux;
+    /// \brief Atm coszen array
+    std::vector<double> atm_coszen;
+    marray<double,1> coszens;
+    /// \brief Atm heights array
+    std::vector<double> atm_heights;
+    marray<double,1> heights;
+    /// \brief Atm energies array
+    std::vector<double> atm_energy;
+    marray<double,1> energies;
+    
+    /// \brief Minimum coszen.
+    double czen_min;
+    /// \brief Maximum coszen.
+    double czen_max;
+    /// \brief number of coszens
+    long unsigned int czen_number;
+    /// \brief Minimum height.
+    double height_min;
+    /// \brief Maximum height.
+    double height_max;
+    /// \brief number of heights
+    long unsigned int height_number;
+    /// \brief Minimum energy.
+    double energy_min;
+    /// \brief Maximum energy.
+    double energy_max;
+    /// \brief number of energies
+    long unsigned int energy_number;
+    /// \brief vector containing interpolators for the differential flux
+    std::vector<TriCubicInterpolator> Interpolators;
+    /// \brief indices for the electron and muon flavors
+    /// defaulted to the first and second flavor states
+    int E_nu_index = 0;
+    int Mu_nu_index = 1;
```

Comment:
I think `E_nu_index` and `Mu_nu_index` should be removed. We should use the usual implicit flavor ordering (nu_e, nu_mu, nu_tau). It would also be good to change all remaining comments an variable names to the standard form (e.g. correct `E_nu` to `nu_e`).